### ISSUE TITLE
openj9-0.18.1 regression vs openj9-0.17.0: GPF Assertion failed at  ... compiler/optimizer/InterpreterEmulator.cpp:208

### ISSUE THREAD

**Issue by avermeer** [eclipse/openj9#8548]
Hello,

Quick feedback on a regression we found when switching from AdoptOpenJDK OpenJ9 11.0.5 (openj9-0.17.0)   to AdoptOpenJDK J9 11.0.6 (openj9-0.18.1) : one of our test consistently fails on both Linux x64 and Windows 64 with the following GPF:

* On Linux x64:
```
Assertion failed at /home/jenkins/workspace/build-scripts/jobs/jdk11u/jdk11u-linux-x64-openj9/workspace/build/src/build/linux-x86_64-normal-server-release/vm/compiler/../compiler/optimizer/InterpreterEmulator.cpp:208: 0
VMState: 0x000501ff
        unexpected bytecode in thunk archetype 0x7fa850e0b670 at bcIndex 40 JBlload2 (33)

compiling jdk/nashorn/internal/scripts/Script$Recompilation$468$5575$\^eval\_.L:1#readUint8#L:1(Ljdk/nashorn/internal/runtime/ScriptFunction;Ljava/lang/Object;)Ljava/lang/Object; at level: warm
#0: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x8aa205) [0x7fa8e1343205]
#1: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x8b6380) [0x7fa8e134f380]
#2: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x5783bb) [0x7fa8e10113bb]
#3: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x5785ad) [0x7fa8e10115ad]
#4: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x26410c) [0x7fa8e0cfd10c]
#5: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x267d82) [0x7fa8e0d00d82]
#6: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x256120) [0x7fa8e0cef120]
#7: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x256f4f) [0x7fa8e0ceff4f]
#8: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x256f4f) [0x7fa8e0ceff4f]
#9: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x256f4f) [0x7fa8e0ceff4f]
#10: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x256f4f) [0x7fa8e0ceff4f]
#11: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x258682) [0x7fa8e0cf1682]
#12: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x23a042) [0x7fa8e0cd3042]
#13: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x245ba9) [0x7fa8e0cdeba9]
#14: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x24d70b) [0x7fa8e0ce670b]
#15: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x666c6e) [0x7fa8e10ffc6e]
#16: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x245036) [0x7fa8e0cde036]
#17: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x728427) [0x7fa8e11c1427]
#18: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x729d8b) [0x7fa8e11c2d8b]
#19: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x51a5fd) [0x7fa8e0fb35fd]
#20: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x184086) [0x7fa8e0c1d086]
#21: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x184ec1) [0x7fa8e0c1dec1]
#22: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9prt29.so(+0x20df3) [0x7fa8e2118df3]
#23: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x186b95) [0x7fa8e0c1fb95]
#24: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x18712e) [0x7fa8e0c2012e]
#25: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x182d63) [0x7fa8e0c1bd63]
#26: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x1831b2) [0x7fa8e0c1c1b2]
#27: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x18325a) [0x7fa8e0c1c25a]
#28: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9prt29.so(+0x20df3) [0x7fa8e2118df3]
#29: /home/data/acme/jdk-11.0.6+10/lib/compressedrefs/libj9jit29.so(+0x1836b4) [0x7fa8e0c1c6b4]

```

* On Windows x64, the output seems to include more details:

```
Assertion failed at C:\cygwin64\tmp\openjdk-jdk11u-windows-x64-openj9\workspace\build\src\build\windows-x86_64-normal-server-release\vm\compiler\optimizer\InterpreterEmulator.cpp:208: 0
VMState: 0x000501ff
	unexpected bytecode in thunk archetype 00007FF623BC3050 at bcIndex 40 JBlload2 (33)

compiling jdk/nashorn/internal/scripts/Script$Recompilation$468$5575$\^eval\_.L:1#readUint8#L:1(Ljdk/nashorn/internal/runtime/ScriptFunction;Ljava/lang/Object;)Ljava/lang/Object; at level: warm

Unhandled exception
Type=Segmentation error vmState=0x000501ff
Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00000000525D6759 ContextFlags=0010005f
Handler1=00000000522CBE60 Handler2=00007FFE6AB0F180 InaccessibleWriteAddress=0000000000000000
RDI=000000E8C0BF4370 RSI=0000000000000021 RAX=00007FFE91D9F180 RBX=000000E8B70A32E4
RCX=0000000052BD1958 RDX=0000000000000000 R8=0000000052BD1958 R9=0000000000000004
R10=0000000000000069 R11=0000000000000288 R12=0000000052240000 R13=00007FF627CB3BD0
R14=0000000000000021 R15=00007FF6279F8FA0
RIP=00000000525D6759 RSP=000000E8C0BF2FC0 RBP=00007FF6279D9CE0 GS=002B
FS=0053 ES=002B DS=002B
XMM0 0000000200000000 (f: 0.000000, d: 4.243992e-314)
XMM1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM2 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM3 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM7 00000000461c4000 (f: 1176256512.000000, d: 5.811479e-315)
XMM8 000000003ecccccd (f: 1053609152.000000, d: 5.205521e-315)
XMM9 000000003dcccccd (f: 1036831936.000000, d: 5.122630e-315)
XMM10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
Module=E:\WS\avejava\win_b64\code\jre\currentjre\bin\compressedrefs\j9jit29.dll
Module_base_address=0000000052240000 Offset_in_DLL=0000000000396759

Method_being_compiled=jdk/nashorn/internal/scripts/Script$Recompilation$458$7215$\^eval\_.L:1#readString#L:1(Ljdk/nashorn/internal/runtime/ScriptFunction;Ljava/lang/Object;)Ljava/lang/Object;
Target=2_90_20200122_442 (Windows Server 2012 R2 6.3 build 9600)
CPU=amd64 (8 logical CPUs) (0x3fb25d000 RAM)
----------- Stack Backtrace -----------
Unhandled exception
Type=Segmentation error vmState=0x000501ff
Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00000000525D6759 ContextFlags=0010005f
Handler1=00000000522CBE60 Handler2=00007FFE6AB0F180 InaccessibleWriteAddress=0000000000000000
RDI=000000E8C0FF42D0 RSI=0000000000000021 RAX=00000000800001C8 RBX=000000E8B70A3364
RCX=00000000FFFFFFFF RDX=000000E8B70A3360 R8=0000000000000004 R9=00000000000000D0
R10=0000000000000000 R11=000000E8C0FF2E00 R12=0000000052240000 R13=00007FF626C30610
R14=0000000000000021 R15=00007FF6269F8FA0
RIP=00000000525D6759 RSP=000000E8C0FF2F20 RBP=00007FF626AB6930 GS=002B
FS=0053 ES=002B DS=002B
XMM0 0000000200000000 (f: 0.000000, d: 4.243992e-314)
XMM1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM2 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM3 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM7 00000000461c4000 (f: 1176256512.000000, d: 5.811479e-315)
XMM8 000000003ecccccd (f: 1053609152.000000, d: 5.205521e-315)
XMM9 000000003dcccccd (f: 1036831936.000000, d: 5.122630e-315)
XMM10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
Module=E:\WS\avejava\win_b64\code\jre\currentjre\bin\compressedrefs\j9jit29.dll
Module_base_address=0000000052240000 Offset_in_DLL=0000000000396759

Method_being_compiled=jdk/nashorn/internal/scripts/Script$Recompilation$471$6555$\^eval\_.L:1#readUint64#L:1(Ljdk/nashorn/internal/runtime/ScriptFunction;Ljava/lang/Object;)Ljava/lang/Object;
Target=2_90_20200122_442 (Windows Server 2012 R2 6.3 build 9600)
CPU=amd64 (8 logical CPUs) (0x3fb25d000 RAM)
----------- Stack Backtrace -----------
Unhandled exception
Type=SegmeAssertion failed at C:\cygwin64\tmp\openjdk-jdk11u-windows-x64-openj9\workspace\build\src\build\windows-x86_64-normal-server-release\vm\compiler\optimizer\InterpreterEmulator.cpp:208: 0
VMState: 0x000501ff
	unexpected bytecode in thunk archetype 00007FF618AFEA50 at bcIndex 40 JBlload2 (33)

compiling jdk/nashorn/internal/scripts/Script$Recompilation$451$4925A$\^eval\_.L:1#BinaryReader(Ljdk/nashorn/internal/runtime/ScriptFunction;Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object; at level: warm

ntation error vmState=0x000501ff
Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00000000525D6759 ContextFlags=0010005f
Handler1=00000000522CBE60 Handler2=00007FFE6AB0F180 InaccessibleWriteAddress=0000000000000000
RDI=000000E8C17F4520 RSI=0000000000000021 RAX=00000000800001C9 RBX=000000E8B77F3014
RCX=000000000000000F RDX=0000000000000004 R8=000000E8C17F3250 R9=00000000000000D0
R10=0000000000000000 R11=000000E8C17F3050 R12=0000000052240000 R13=00007FF623BC3050
R14=0000000000000021 R15=00007FF623A38F20
RIP=00000000525D6759 RSP=000000E8C17F3170 RBP=00007FF623A73690 GS=002B
FS=0053 ES=002B DS=002B
XMM0 0000000200000000 (f: 0.000000, d: 4.243992e-314)
XMM1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM2 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM3 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM7 00000000461c4000 (f: 1176256512.000000, d: 5.811479e-315)
XMM8 000000003ecccccd (f: 1053609152.000000, d: 5.205521e-315)
XMM9 000000003dcccccd (f: 1036831936.000000, d: 5.122630e-315)
XMM10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
Module=E:\WS\avejava\win_b64\code\jre\currentjre\bin\compressedrefs\j9jit29.dll
Module_base_address=0000000052240000 Offset_in_DLL=0000000000396759

Method_being_compiled=jdk/nashorn/internal/scripts/Script$Recompilation$468$5575$\^eval\_.L:1#readUint8#L:1(Ljdk/nashorn/internal/runtime/ScriptFunction;Ljava/lang/Object;)Ljava/lang/Object;
Target=2_90_20200122_442 (Windows Server 2012 R2 6.3 build 9600)
CPU=amd64 (8 logical CPUs) (0x3fb25d000 RAM)
----------- Stack Backtrace -----------
Unhandled exception
Type=Segmentation error vmState=0x000501ff
Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00000000525D6759 ContextFlags=0010005f
Handler1=00000000522CBE60 Handler2=00007FFE6AB0F180 InaccessibleWriteAddress=0000000000000000
RDI=000000E8C23F2D10 RSI=0000000000000021 RAX=00000000800001C9 RBX=000000E8B77F2E14
RCX=000000000000000F RDX=0000000000000004 R8=000000E8C23F1A40 R9=00000000000000D0
R10=0000000000000000 R11=000000E8C23F1840 R12=0000000052240000 R13=00007FF618AFEA50
R14=0000000000000021 R15=00007FF618A28F20
RIP=00000000525D6759 RSP=000000E8C23F1960 RBP=00007FF618A327A0 GS=002B
FS=0053 ES=002B DS=002B
XMM0 0000000200000000 (f: 0.000000, d: 4.243992e-314)
XMM1 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM2 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM3 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM7 00000000461c4000 (f: 1176256512.000000, d: 5.811479e-315)
XMM8 000000003ecccccd (f: 1053609152.000000, d: 5.205521e-315)
XMM9 000000003dcccccd (f: 1036831936.000000, d: 5.122630e-315)
XMM10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
Module=E:\WS\avejava\win_b64\code\jre\currentjre\bin\compressedrefs\j9jit29.dll
Module_base_address=0000000052240000 Offset_in_DLL=0000000000396759

Method_being_compiled=jdk/nashorn/internal/scripts/Script$Recompilation$451$4925A$\^eval\_.L:1#BinaryReader(Ljdk/nashorn/internal/runtime/ScriptFunction;Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;
Target=2_90_20200122_442 (Windows Server 2012 R2 6.3 build 9600)
CPU=amd64 (8 logical CPUs) (0x3fb25d000 RAM)
----------- Stack Backtrace -----------
Java_java_lang_invoke_MutableCallSite_invalidate+0x2ccdb9 (0x00000000525D6759 [j9jit29+0x396759])
Java_java_lang_invoke_MutableCallSite_invalidate+0x2cd0e8 (0x00000000525D6A88 [j9jit29+0x396a88])
Java_java_lang_invoke_MutableCallSite_invalidate+0x2ccdf8 (0x00000000525D6798 [j9jit29+0x396798])
Java_java_lang_invoke_MutableCallSite_invalidate+0x7e296 (0x0000000052387C36 [j9jit29+0x147c36])
Java_java_lang_invoke_MutableCallSite_invalidate+0x7db6f (0x000000005238750F [j9jit29+0x14750f])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73307 (0x000000005237CCA7 [j9jit29+0x13cca7])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x586f8 (0x0000000052362098 [j9jit29+0x122098])
Java_java_lang_invoke_MutableCallSite_invalidate+0x5bd28 (0x00000000523656C8 [j9jit29+0x1256c8])
Java_java_lang_invoke_MutableCallSite_invalidate+0x5939b (0x0000000052362D3B [j9jit29+0x122d3b])
Java_java_lang_invoke_MutableCallSite_invalidate+0x3733e1 (0x000000005267CD81 [j9jit29+0x43cd81])
Java_java_lang_invoke_MutableCallSite_invalidate+0x58c43 (0x00000000523625E3 [j9jit29+0x1225e3])
Java_java_lang_invoke_MutableCallSite_invalidate+0x41e319 (0x0000000052727CB9 [j9jit29+0x4e7cb9])
Java_java_lang_invoke_MutableCallSite_invalidate+0x41ac85 (0x0000000052724625 [j9jit29+0x4e4625])
Java_java_lang_invoke_MutableCallSite_invalidate+0x269f9f (0x000000005257393F [j9jit29+0x33393f])
(0x00000000522B8C6A [j9jit29+0x78c6a])
(0x00000000522BBCB2 [j9jit29+0x7bcb2])
j9port_isCompatible+0x1ceb6 (0x00007FFE6AB0FBC6 [j9prt29+0x1fbc6])
j9port_isCompatible+0x1e5b4 (0x00007FFE6AB112C4 [j9prt29+0x212c4])
(0x00000000522B83FE [j9jit29+0x783fe])
(0x00000000522BDFE2 [j9jit29+0x7dfe2])
(0x00000000522BD9CA [j9jit29+0x7d9ca])
(0x00000000522CBBEF [j9jit29+0x8bbef])
j9port_isCompatible+0x1e5ef (0x00007FFE6AB112FF [j9prt29+0x212ff])
(0x00000000522CB80D [j9jit29+0x8b80d])
omrthread_get_category+0xa42 (0x00007FFE7B924452 [J9THR29+0x4452])
_o__realloc_base+0x60 (0x00007FFE87A4F4A0 [ucrtbase+0x1f4a0])
BaseThreadInitThunk+0x22 (0x00007FFE91BD13D2 [KERNEL32+0x13d2])
RtlUserThreadStart+0x34 (0x00007FFE91D454F4 [ntdll+0x154f4])
---------------------------------------
Java_java_lang_invoke_MutableCallSite_invalidate+0x2ccdb9 (0x00000000525D6759 [j9jit29+0x396759])
Java_java_lang_invoke_MutableCallSite_invalidate+0x2cd0e8 (0x00000000525D6A88 [j9jit29+0x396a88])
Java_java_lang_invoke_MutableCallSite_invalidate+0x2ccdf8 (0x00000000525D6798 [j9jit29+0x396798])
Java_java_lang_invoke_MutableCallSite_invalidate+0x7e296 (0x0000000052387C36 [j9jit29+0x147c36])
Java_java_lang_invoke_MutableCallSite_invalidate+0x7db6f (0x000000005238750F [j9jit29+0x14750f])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73307 (0x000000005237CCA7 [j9jit29+0x13cca7])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x586f8 (0x0000000052362098 [j9jit29+0x122098])
Java_java_lang_invoke_MutableCallSite_invalidate+0x5bd28 (0x00000000523656C8 [j9jit29+0x1256c8])
Java_java_lang_invoke_MutableCallSite_invalidate+0x5939b (0x0000000052362D3B [j9jit29+0x122d3b])
Java_java_lang_invoke_MutableCallSite_invalidate+0x3733e1 (0x000000005267CD81 [j9jit29+0x43cd81])
Java_java_lang_invoke_MutableCallSite_invalidate+0x58c43 (0x00000000523625E3 [j9jit29+0x1225e3])
Java_java_lang_invoke_MutableCallSite_invalidate+0x41e319 (0x0000000052727CB9 [j9jit29+0x4e7cb9])
Java_java_lang_invoke_MutableCallSite_invalidate+0x41ac85 (0x0000000052724625 [j9jit29+0x4e4625])
Java_java_lang_invoke_MutableCallSite_invalidate+0x269f9f (0x000000005257393F [j9jit29+0x33393f])
(0x00000000522B8C6A [j9jit29+0x78c6a])
(0x00000000522BBCB2 [j9jit29+0x7bcb2])
j9port_isCompatible+0x1ceb6 (0x00007FFE6AB0FBC6 [j9prt29+0x1fbc6])
j9port_isCompatible+0x1e5b4 (0x00007FFE6AB112C4 [j9prt29+0x212c4])
(0x00000000522B83FE [j9jit29+0x783fe])
(0x00000000522BDFE2 [j9jit29+0x7dfe2])
(0x00000000522BD9CA [j9jit29+0x7d9ca])
(0x00000000522CBBEF [j9jit29+0x8bbef])
j9port_isCompatible+0x1e5ef (0x00007FFE6AB112FF [j9prt29+0x212ff])
(0x00000000522CB80D [j9jit29+0x8b80d])
omrthread_get_category+0xa42 (0x00007FFE7B924452 [J9THR29+0x4452])
_o__realloc_base+0x60 (0x00007FFE87A4F4A0 [ucrtbase+0x1f4a0])
BaseThreadInitThunk+0x22 (0x00007FFE91BD13D2 [KERNEL32+0x13d2])
RtlUserThreadStart+0x34 (0x00007FFE91D454F4 [ntdll+0x154f4])
---------------------------------------
Java_java_lang_invoke_MutableCallSite_invalidate+0x2ccdb9 (0x00000000525D6759 [j9jit29+0x396759])
Java_java_lang_invoke_MutableCallSite_invalidate+0x2cd0e8 (0x00000000525D6A88 [j9jit29+0x396a88])
Java_java_lang_invoke_MutableCallSite_invalidate+0x2ccdf8 (0x00000000525D6798 [j9jit29+0x396798])
Java_java_lang_invoke_MutableCallSite_invalidate+0x7e296 (0x0000000052387C36 [j9jit29+0x147c36])
Java_java_lang_invoke_MutableCallSite_invalidate+0x7db6f (0x000000005238750F [j9jit29+0x14750f])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73307 (0x000000005237CCA7 [j9jit29+0x13cca7])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x586f8 (0x0000000052362098 [j9jit29+0x122098])
Java_java_lang_invoke_MutableCallSite_invalidate+0x5bd28 (0x00000000523656C8 [j9jit29+0x1256c8])
Java_java_lang_invoke_MutableCallSite_invalidate+0x5939b (0x0000000052362D3B [j9jit29+0x122d3b])
Java_java_lang_invoke_MutableCallSite_invalidate+0x3733e1 (0x000000005267CD81 [j9jit29+0x43cd81])
Java_java_lang_invoke_MutableCallSite_invalidate+0x58c43 (0x00000000523625E3 [j9jit29+0x1225e3])
Java_java_lang_invoke_MutableCallSite_invalidate+0x41e319 (0x0000000052727CB9 [j9jit29+0x4e7cb9])
Java_java_lang_invoke_MutableCallSite_invalidate+0x41ac85 (0x0000000052724625 [j9jit29+0x4e4625])
Java_java_lang_invoke_MutableCallSite_invalidate+0x269f9f (0x000000005257393F [j9jit29+0x33393f])
(0x00000000522B8C6A [j9jit29+0x78c6a])
(0x00000000522BBCB2 [j9jit29+0x7bcb2])
j9port_isCompatible+0x1ceb6 (0x00007FFE6AB0FBC6 [j9prt29+0x1fbc6])
j9port_isCompatible+0x1e5b4 (0x00007FFE6AB112C4 [j9prt29+0x212c4])
(0x00000000522B83FE [j9jit29+0x783fe])
(0x00000000522BDFE2 [j9jit29+0x7dfe2])
(0x00000000522BD9CA [j9jit29+0x7d9ca])
(0x00000000522CBBEF [j9jit29+0x8bbef])
j9port_isCompatible+0x1e5ef (0x00007FFE6AB112FF [j9prt29+0x212ff])
(0x00000000522CB80D [j9jit29+0x8b80d])
omrthread_get_category+0xa42 (0x00007FFE7B924452 [J9THR29+0x4452])
_o__realloc_base+0x60 (0x00007FFE87A4F4A0 [ucrtbase+0x1f4a0])
BaseThreadInitThunk+0x22 (0x00007FFE91BD13D2 [KERNEL32+0x13d2])
RtlUserThreadStart+0x34 (0x00007FFE91D454F4 [ntdll+0x154f4])
---------------------------------------
Java_java_lang_invoke_MutableCallSite_invalidate+0x2ccdb9 (0x00000000525D6759 [j9jit29+0x396759])
Java_java_lang_invoke_MutableCallSite_invalidate+0x2cd0e8 (0x00000000525D6A88 [j9jit29+0x396a88])
Java_java_lang_invoke_MutableCallSite_invalidate+0x2ccdf8 (0x00000000525D6798 [j9jit29+0x396798])
Java_java_lang_invoke_MutableCallSite_invalidate+0x7e296 (0x0000000052387C36 [j9jit29+0x147c36])
Java_java_lang_invoke_MutableCallSite_invalidate+0x7db6f (0x000000005238750F [j9jit29+0x14750f])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73307 (0x000000005237CCA7 [j9jit29+0x13cca7])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x73d36 (0x000000005237D6D6 [j9jit29+0x13d6d6])
Java_java_lang_invoke_MutableCallSite_invalidate+0x707c0 (0x000000005237A160 [j9jit29+0x13a160])
Java_java_lang_invoke_MutableCallSite_invalidate+0x586f8 (0x0000000052362098 [j9jit29+0x122098])
Java_java_lang_invoke_MutableCallSite_invalidate+0x5bd28 (0x00000000523656C8 [j9jit29+0x1256c8])
Java_java_lang_invoke_MutableCallSite_invalidate+0x5939b (0x0000000052362D3B [j9jit29+0x122d3b])
Java_java_lang_invoke_MutableCallSite_invalidate+0x3733e1 (0x000000005267CD81 [j9jit29+0x43cd81])
Java_java_lang_invoke_MutableCallSite_invalidate+0x58c43 (0x00000000523625E3 [j9jit29+0x1225e3])
Java_java_lang_invoke_MutableCallSite_invalidate+0x41e319 (0x0000000052727CB9 [j9jit29+0x4e7cb9])
Java_java_lang_invoke_MutableCallSite_invalidate+0x41ac85 (0x0000000052724625 [j9jit29+0x4e4625])
Java_java_lang_invoke_MutableCallSite_invalidate+0x269f9f (0x000000005257393F [j9jit29+0x33393f])
(0x00000000522B8C6A [j9jit29+0x78c6a])
(0x00000000522BBCB2 [j9jit29+0x7bcb2])
j9port_isCompatible+0x1ceb6 (0x00007FFE6AB0FBC6 [j9prt29+0x1fbc6])
j9port_isCompatible+0x1e5b4 (0x00007FFE6AB112C4 [j9prt29+0x212c4])
(0x00000000522B83FE [j9jit29+0x783fe])
(0x00000000522BDFE2 [j9jit29+0x7dfe2])
(0x00000000522BD9CA [j9jit29+0x7d9ca])
(0x00000000522CBBEF [j9jit29+0x8bbef])
j9port_isCompatible+0x1e5ef (0x00007FFE6AB112FF [j9prt29+0x212ff])
(0x00000000522CB80D [j9jit29+0x8b80d])
omrthread_get_category+0xa42 (0x00007FFE7B924452 [J9THR29+0x4452])
_o__realloc_base+0x60 (0x00007FFE87A4F4A0 [ucrtbase+0x1f4a0])
BaseThreadInitThunk+0x22 (0x00007FFE91BD13D2 [KERNEL32+0x13d2])
RtlUserThreadStart+0xAssertion failed at C:\cygwin64\tmp\openjdk-jdk11u-windows-x64-openj9\workspace\build\src\build\windows-x86_64-normal-server-release\vm\compiler\optimizer\InterpreterEmulator.cpp:208: 0
VMState: 0x000501ff
	unexpected bytecode in thunk archetype 00007FF625D35D40 at bcIndex 40 JBlload2 (33)

compiling jdk/nashorn/internal/scripts/Script$Recompilation$458$7215$\^eval\_.L:1#readString#L:1(Ljdk/nashorn/internal/runtime/ScriptFunction;Ljava/lang/Object;)Ljava/lang/Object; at level: warm

34 (0x00007FFE91D454F4 [ntdll+0x154f4])
---------------------------------------
JVMDUMP039I Processing dump event "gpf", detail "" at 2020/02/10 18:14:25 - please wait.
JVMDUMP039I Processing dump event "gpf", detail "" at 2020/02/10 18:14:25 - please wait.
JVMDUMP039I Processing dump event "gpf", detail "" at 2020/02/10 18:14:25 - please wait.
JVMDUMP039I Processing dump event "gpf", detail "" at 2020/02/10 18:14:25 - please wait.
JVMDUMP032I JVM requested System dump using 'C:\temp\ADL_ODT_6424\MKODT_DATA\core.20200210.181425.5788.0001.dmp' in response to an event
JVMDUMP010I System dump written to C:\temp\ADL_ODT_6424\MKODT_DATA\core.20200210.181425.5788.0001.dmp
JVMDUMP032I JVM requested Java dump using 'C:\temp\ADL_ODT_6424\MKODT_DATA\javacore.20200210.181425.5788.0005.txt' in response to an event
JVMDUMP032I JVM requested System dump using 'C:\temp\ADL_ODT_6424\MKODT_DATA\core.20200210.181425.5788.0002.dmp' in response to an event
JVMDUMP010I System dump written to C:\temp\ADL_ODT_6424\MKODT_DATA\core.20200210.181425.5788.0002.dmp
JVMDUMP032I JVM requested Java dump using 'C:\temp\ADL_ODT_6424\MKODT_DATA\javacore.20200210.181425.5788.0006.txt' in response to an event
JVMDUMP010I Java dump written to C:\temp\ADL_ODT_6424\MKODT_DATA\javacore.20200210.181425.5788.0005.txt
JVMDUMP032I JVM requested Snap dump using 'C:\temp\ADL_ODT_6424\MKODT_DATA\Snap.20200210.181425.5788.0007.trc' in response to an event
JVMDUMP010I Snap dump written to C:\temp\ADL_ODT_6424\MKODT_DATA\Snap.20200210.181425.5788.0007.trc
JVMDUMP032I JVM requested System dump using 'C:\temp\ADL_ODT_6424\MKODT_DATA\core.20200210.181425.5788.0003.dmp' in response to an event
JVMDUMP010I Java dump written to C:\temp\ADL_ODT_6424\MKODT_DATA\javacore.20200210.181425.5788.0006.txt
JVMDUMP010I System dump written to C:\temp\ADL_ODT_6424\MKODT_DATA\core.20200210.181425.5788.0003.dmp
JVMDUMP032I JVM requested Java dump using 'C:\temp\ADL_ODT_6424\MKODT_DATA\javacore.20200210.181425.5788.0010.txt' in response to an event
JVMDUMP007I JVM Requesting JIT dump using 'C:\temp\ADL_ODT_6424\MKODT_DATA\jitdump.20200210.181425.5788.0008.dmp'
```

I wanted to check if the problem can be reproduced with a more recent OpenJ9 version, but as of today the AdoptOpenJDK 11 OpenJ9 nightly builds are missing both Linux x64 and Windows x64 binaries, so I cannot do this test. If there's a way to get a recent set of binaries for either Linux x64 or Windows x64, then I'll be happy to check our program with these.

It will take a while before we can extract the code that triggered this issue to submit a sample, but I felt that this is nevertheless important for the OpenJ9 community to make this first & quick report.

Kind regards,
Alexandre Vermeerbergen

**Comment by DanHeidinga** [comment]
```
unexpected bytecode in thunk archetype 0x7fa850e0b670 at bcIndex 40 JBlload2 
```

This looks very similar to https://github.com/eclipse/openj9/pull/8512 which was merged into master a couple of days ago.

We'll follow up on the Adopt JDK11 builds as they should be happening

**Comment by DanHeidinga** [comment]
fyi @cathyzhyi

**Comment by avermeer** [comment]
Thanks! if confirmed that a fix is already available, then how about a 0.18.2 ? 
Impact is rather large on our side, and staying on 0.17.0 prevents us from getting Jan 2020 security fixes...

**Comment by DanHeidinga** [comment]
> Thanks! if confirmed that a fix is already available, then how about a 0.18.2 ?

This would be something we'd have to discuss on the OpenJ9 community call.  The call runs on Wednesdays @ 11 am ET with details are posted to the #planning channel in Slack or in the agendas in https://drive.google.com/drive/folders/15FN_IVizGzDVurOqkgHI68F-kTF9ssoG

**Comment by DanHeidinga** [comment]
@cathyzhyi is there any way to disable the InterpreterEmulator to work around this issue?

**Comment by cathyzhyi** [comment]
> @cathyzhyi is there any way to disable the InterpreterEmulator to work around this issue?

@DanHeidinga yes, there is -Xjit:disableMethodHandleInlineWithoutPeeking

**Comment by DanHeidinga** [comment]
@avermeer Are you able to test with `-Xjit:disableMethodHandleInlineWithoutPeeking` to confirm that the problem no longer occurs with that option?

**Comment by JasonFengJ9** [comment]
A similar failure at `Java_java_lang_invoke_MutableCallSite_invalidate+0xf0832 (0x00000000716A5F12 [j9jit29+0x185f12])` from https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_windows_Nightly/840/tapResults/
```
LT  00:30:43.595 - Starting thread. Suite=0 thread=6
LT  stderr Unhandled exception
LT  stderr Type=Segmentation error vmState=0x00000000
LT  stderr Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00000000716A5F12 ContextFlags=0010005f
LT  stderr Handler1=00007FFAE0D5CBC0 Handler2=00007FFAE0F7C910 InaccessibleReadAddress=FFFFFFFFFFFFFFFF
LT  stderr RDI=000000005E6ADD63 RSI=000000005E6ADD68 RAX=E03FBCB8E0409FF0 RBX=00000000FFE5C520
LT  stderr RCX=00000000FFF22F00 RDX=000000005E6ADD68 R8=00000000FFFFFFFF R9=00000000FFE5BC48
LT  stderr R10=000000000255F100 R11=00000000FFE5BC48 R12=0000000000000004 R13=0000000000000004
LT  stderr R14=0000000000000000 R15=00000000E0411FD0
LT  stderr RIP=00000000716A5F12 RSP=000000000214BEA0 RBP=0000000002120000 GS=002B
LT  stderr FS=0053 ES=002B DS=002B
LT  stderr XMM0 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM1 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM2 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM3 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM4 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM5 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM6 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM7 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM8 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM9 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM10 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM11 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM12 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM13 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM14 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr XMM15 0000000000000000 (f: 0.000000, d: 0.000000e+000)
LT  stderr Module=C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly\openjdkbinary\j2sdk-image\jre\bin\compressedrefs\j9jit29.dll
LT  stderr Module_base_address=0000000071520000 Offset_in_DLL=0000000000185f12
LT  stderr Target=2_90_20200212_297 (Windows Server 2012 R2 6.3 build 9600)
LT  stderr CPU=amd64 (8 logical CPUs) (0x1ffb9c000 RAM)
LT  stderr ----------- Stack Backtrace -----------
LT  stderr Java_java_lang_invoke_MutableCallSite_invalidate+0xf0832 (0x00000000716A5F12 [j9jit29+0x185f12])
LT  stderr (0x00000000FFF22F00)
LT  stderr (0x00000000FFF22F00)
LT  stderr (0x0000000000000001)
LT  stderr (0x00000000FFC4D5A0)
LT  stderr ---------------------------------------
LT  stderr JVMDUMP039I Processing dump event "gpf", detail "" at 2020/02/13 00:31:01 - please wait.
LT  stderr JVMDUMP032I JVM requested System dump using 'C:\Users\jenkins\workspace\Test_openjdk8_j9_special.system_x86-64_windows_Nightly\openjdk-tests\TKG\test_output_15815743107888\MauveMultiThreadLoadTest_special_22\20200213-003029-MauveMultiThreadLoadTest\results\core.20200213.003101.4272.0001.dmp' in response to an event
```
To rebuild the failed tests in <Jenkins URL>=https://ci.adoptopenjdk.net/job/Grinder, use the following links:
04:08:54  https://ci.adoptopenjdk.net/job/Grinder/parambuild/?JDK_VERSION=8&JDK_IMPL=openj9&BUILD_LIST=system/mauveLoadTest&PLATFORM=x86-64_windows&TARGET=MauveMultiThreadLoadTest_special_22

**Comment by pshipton** [comment]
@JasonFengJ9 there is no indication this is the same problem. The vmstate isn't the same, there is no Method_being_compiled, and the offset from MutableCallSite_invalidate is much different. The only reason Java_java_lang_invoke_MutableCallSite_invalidate is in these stack traces is because there are no debug symbols available. I expect your failure report belongs in https://github.com/eclipse/openj9/issues/8482.

**Comment by JasonFengJ9** [comment]
> I expect your failure report belongs in #8482.

Agreed. I was eager to locate a testcase for this segmentation error .

**Comment by avermeer** [comment]
@DanHeidinga sorry for delayed answer, I was away for the past 3 days.

I ran our application with `-Xjit:disableMethodHandleInlineWithoutPeeking` and I confirm that it is working fine with this.

Now what is the next step? spreading `-Xjit:disableMethodHandleInlineWithoutPeeking` on our side is inconvenient, since we have many different involved shells, and even more test cases of various forms that would need this option to by-pass the crash. And yet, we are still looking to get January 2020 security fixes.

Is a 0.18.2 fixing this crash something which can be considered?

Kind regards,
Alexandre

**Comment by pshipton** [comment]
@avermeer In the JVM installation there is a file `lib/options.default` where you can add the `-Xjit:disableMethodHandleInlineWithoutPeeking` option. You can add it on a new line in the file. Since this work around is available, we're unlikely to create a 0.18.2 release.

**Comment by avermeer** [comment]
Hello @pshipton, thank you very much for this by-pass : worked like charm!

Any hope for a fix in 0.19.0 ?
Anything in nightly builds which I could test against our scenario?

Kind regards,
Alexandre

**Comment by pshipton** [comment]
> Any hope for a fix in 0.19.0 ?

Yes, the problem is resolved in the Milestone 1 builds, and also in the regular nightly builds.
https://github.com/AdoptOpenJDK/openjdk14-binaries/releases/tag/jdk-14%2B34_openj9-0.19.0-m1

**Comment by avermeer** [comment]
Sounds good, but I should have asked more precisely : is there a Java 11 preview with the fix which I can test on Linux x64 and Windows x64?

On AdoptOpenJDK site, the nightly builds are still missing these 2 operating systems:
https://adoptopenjdk.net/nightly.html?variant=openjdk11&jvmVariant=openj9
and "latest" nightly build is old (7th of February...)

**Comment by pshipton** [comment]
I see https://github.com/AdoptOpenJDK/openjdk-build/issues/1544 is open for the missing xLinux builds, can you please update it or open a new Issue for the missing Windows builds.

**Comment by pshipton** [comment]
While they aren't recommended for use, there are nightly builds at the OpenJ9 project. Last night's build is https://ci.eclipse.org/openj9/view/Pipelines/job/Pipeline-Build-Test-All-11/78/
If you click on the build link for the platform, the JVM used for testing is attached as an artifact.

**Comment by avermeer** [comment]
I have updated  AdoptOpenJDK/openjdk-build#1544 and I will wait for the nightly images to be available at AdoptOpenJDK site.

**Comment by avermeer** [comment]
Ahem... latest available nightly build of  AdoptOpenJDK OpenJ9 is still the one of Feb 7th, and still no Linux x64 nor Windows x64 image is available, see https://adoptopenjdk.net/nightly.html?variant=openjdk11&jvmVariant=openj9 

Is that a sign that AdoptOpenJDK support for OpenJ9 is fading away? (hope not)

**Comment by pshipton** [comment]
OpenJ9 is not fading away. Adopt is having build problems. The builds seem to be available directly from jenkins, just not from the website. https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-linux-x64-openj9/

**Comment by pshipton** [comment]
and https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk11u/job/jdk11u-windows-x64-openj9/

**Comment by JasonFengJ9** [comment]
A latest internal build `openjdk11_j9_sanity.openjdk_ppc64le_linux_Nightly` has a similar failure with same  `vmState=0x000501ff`.
```
00:20:03  Unhandled exception
00:20:03  Type=Segmentation error vmState=0x000501ff
00:20:03  J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
00:20:03  Handler1=00003FFFA62E7900 Handler2=00003FFFA614F740
00:20:03  R0=00003FFFA492B4F4 R1=00003FFF86B705B0 R2=00003FFFA5675200 R3=6E692F676E616C2F
00:20:03  R4=0000000000000003 R5=00003FFF0E0C31D8 R6=000000000000001E R7=00003FFF0E0C3470
00:20:03  R8=00003FFFA492B05C R9=0000000000000018 R10=6E692F676E616C2F R11=00003FFF86B70308
00:20:03  R12=0000000028222288 R13=00003FFF86B86900 R14=00003FFFA5640DD0 R15=00003FFF86B71978
00:20:03  R16=00003FFF0E000000 R17=0000000000079300 R18=000000000000002E R19=00003FFF0E0D29E0
00:20:03  R20=0000000000000000 R21=0000000000000000 R22=00003FFF86B719F8 R23=0000000000000001
00:20:03  R24=0000000000000037 R25=0000000000000001 R26=00003FFF0E000000 R27=00003FFF0E0C3600
00:20:03  R28=00003FFFA53B4F80 R29=0000000000000003 R30=00003FFF86B719F8 R31=6E692F676E616C2F
00:20:03  NIP=00003FFFA4F734A8 MSR=800000010280F033 ORIG_GPR3=00003FFFA4865170 CTR=00003FFFA492B4E4
00:20:03  LINK=00003FFFA492A8B8 XER=0000000000000000 CCR=0000000048222284 SOFTE=0000000000000001
00:20:03  TRAP=0000000000000300 DAR=6E692F676E616C47 dsisr=0000000040000000 RESULT=0000000000000000
00:20:03  FPR0 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR1 3ff0000000000000 (f: 0.000000, d: 1.000000e+00)
00:20:03  FPR2 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR3 00003fffa57d8180 (f: 2776465920.000000, d: 3.476603e-310)
00:20:03  FPR4 00003fffa57de140 (f: 2776490240.000000, d: 3.476603e-310)
00:20:03  FPR5 00003fffa57d9900 (f: 2776471808.000000, d: 3.476603e-310)
00:20:03  FPR6 6e616c2f6176616a (f: 1635148160.000000, d: 5.038250e+223)
00:20:03  FPR7 646e61487261562f (f: 1918981632.000000, d: 6.011120e+175)
00:20:03  FPR8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR10 40c3880000000000 (f: 0.000000, d: 1.000000e+04)
00:20:03  FPR11 3fee666666666666 (f: 1717986944.000000, d: 9.500000e-01)
00:20:03  FPR12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR13 00003fffa57d9270 (f: 2776470016.000000, d: 3.476603e-310)
00:20:03  FPR14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR16 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR17 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR18 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR19 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR20 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR21 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR22 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR23 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR24 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR25 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR26 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR27 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR28 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR29 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR30 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  FPR31 0000000000000000 (f: 0.000000, d: 0.000000e+00)
00:20:03  Module=/home/jenkins/workspace/Test_openjdk11_j9_sanity.openjdk_ppc64le_linux_Nightly/openjdkbinary/j2sdk-image/lib/compressedrefs/libj9jit29.so
00:20:03  Module_base_address=00003FFFA4650000
00:20:03  
00:20:03  Method_being_compiled=java/lang/invoke/BruteArgumentMoverHandle.invokeExact_thunkArchetype_X(I)I
00:20:03  Target=2_90_20200225_329 (Linux 4.4.0-142-generic)
00:20:03  CPU=ppc64le (4 logical CPUs) (0x3fdba0000 RAM)
00:20:03  ----------- Stack Backtrace -----------
00:20:03  (0x00003FFFA4F734A8 [libj9jit29.so+0x9234a8])
00:20:03  (0x00003FFF86B70650 [<unknown>+0x0])
00:20:03  (0x00003FFFA492B4F4 [libj9jit29.so+0x2db4f4])
00:20:03  (0x00003FFFA49307C4 [libj9jit29.so+0x2e07c4])
00:20:03  (0x00003FFFA491CE44 [libj9jit29.so+0x2cce44])
00:20:03  (0x00003FFFA491F404 [libj9jit29.so+0x2cf404])
00:20:03  (0x00003FFFA491DCA0 [libj9jit29.so+0x2cdca0])
00:20:03  (0x00003FFFA491F404 [libj9jit29.so+0x2cf404])
00:20:03  (0x00003FFFA48FA7D8 [libj9jit29.so+0x2aa7d8])
00:20:03  (0x00003FFFA49095F0 [libj9jit29.so+0x2b95f0])
00:20:03  (0x00003FFFA491268C [libj9jit29.so+0x2c268c])
00:20:03  (0x00003FFFA4DF1418 [libj9jit29.so+0x7a1418])
00:20:03  (0x00003FFFA4908540 [libj9jit29.so+0x2b8540])
00:20:03  (0x00003FFFA4EC883C [libj9jit29.so+0x87883c])
```

To rebuild the failed tests in <Jenkins URL>=https://ci.adoptopenjdk.net/job/Grinder, use the following links:
01:53:15  https://ci.adoptopenjdk.net/job/Grinder/parambuild/?JDK_VERSION=11&JDK_IMPL=openj9&BUILD_LIST=openjdk&PLATFORM=ppc64le_linux&TARGET=jdk_lang_0

@cathyzhyi is this a related failure or a new issue should be opened instead?

**Comment by pshipton** [comment]
@JasonFengJ9 looks like a dup of https://github.com/eclipse/openj9/issues/8661

**Comment by pshipton** [comment]
This should be completely fixed by https://github.com/eclipse/openj9/pull/8598. In advance of the 0.20.0 release in April, it can be verified by using a Nightly build from AdoptOpenJDK.net.
